devices/Ft6xx6x/samples/Program.cs

```diff
@@ -28,7 +28,6 @@
 Debug.WriteLine($"Monitor mode: {sensor.MonitorModeEnabled}");
 Debug.WriteLine($"Proximity sensing: {sensor.ProximitySensingEnabled}");
 
-gpio.OpenPin(39, PinMode.Input);
```

Comment:
why this needs to be removed?

Reply:
the sample which is for m5core2 throws an exception on openpin because the function initializescreen in pck m5core2 already opens the pin

Reply:
I think this sample is not only for m5core2, so the better way would be to check if pin is already open. Or just use "TryOpenPin" method instead of OpenPin.

Reply:
I will change it to tryopen for clarity instead of removing

Reply:
IsPinOpen added